### PR TITLE
Added basic decimal conversions

### PR DESCRIPTION
**Pull request by Heshan94**
#36

### CODE REVIEW

File: conversions/decimal2Binary.c [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+{
+    long num;
+ 
+    printf("Enter a decimal integer \n");
+    scanf("%ld", &num);
+    decimal2Binary(num);
+
+}
+
+
```
Comment (dynamitechetan) [inline conversions/decimal2Binary.c]: please remove these blank lines

File: conversions/decimal2Hexa.c [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+ printf("Enter decimal number: ");
+
+ scanf("%ld", &decimalnum);
+decimal2Hexadecimal(decimalnum);
+
+ 
+}
+
+
```
Comment (dynamitechetan) [inline conversions/decimal2Hexa.c]: same here.. please remove these blank lines

File: conversions/decimal2Hexa.c [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+
+
+
+
+
+
+
```
Comment (dynamitechetan) [inline conversions/decimal2Hexa.c]: here too. please remove these blank lines.

**Comment by Heshan94** [comment]
removed blank lines and updated codes

File: conversions/decimal2Binary.c [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+	
+    long num;
+ 
+    printf("Enter a decimal integer \n");
+    scanf("%ld", &num);
+    decimal2Binary(num);
+}
+void decimal2Binary(long num){
+
+long  decimal_num, remainder, base = 1, binary = 0, no_of_1s = 0;
```
Comment (sachinarora707) [inline conversions/decimal2Binary.c]: Please be consistent with the indentation for better readability.

**Comment by Heshan94** [comment]
updated codes

File: Conversions/Binary To Decimal.c [hunk 5]
```diff
@@ -1,17 +0,0 @@
...
-	while(number>0) {
-	
-		remainder = number%10;
-		number = number/10;
-		decimal_number += remainder*temp;
-		temp = temp*2;//used as power of 2
-	
-	}
-	printf("%d",decimal_number);
-}
```
Comment (sachinarora707) [inline Conversions/Binary To Decimal.c]: @Heshan94 You've done a great job. But, your main function has return type 'int' but it is not returning anything. Return 0 at the end of the function.

File: conversions/decimal2Binary.c [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+    scanf("%ld", &num);
+    decimal2Binary(num);
+
+return 0;
+} 
+
+/***function for convert decimal numbers to binary numbers****************/
+void decimal2Binary(long num){
+
+long  decimal_num, remainder, base, binary, no_of_1s;
```
Comment (sachinarora707) [inline conversions/decimal2Binary.c]: Please maintain consistency with naming conventions and indentation.

File: conversions/decimal2Hexa.c [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+			quotient = quotient / 16;}
+
+    // print the hexadecimal number
+
+    for (i = j; i >= 0; i--){
+            printf("%c", hexadecimalnum[i]);}
+
+	printf("\n");
+}
```
Comment (sachinarora707) [inline conversions/decimal2Hexa.c]: Leave one blank line at the end of file.

**Pull request merged**